lib/versioned/^3.7.0/index.js

```diff
@@ -34,20 +37,56 @@ function execute(opts, env, config) {
   stdout.unmute();
 
   process.nextTick(function() {
+    var tree;
+
     if (opts.tasksSimple) {
       return logTasksSimple(env, gulpInst);
     }
     if (opts.tasks) {
-      var tree = taskTree(gulpInst.tasks);
+      tree = taskTree(gulpInst.tasks);
       if (config.description && isString(config.description)) {
         tree.label = config.description;
       } else {
         tree.label = 'Tasks for ' + chalk.magenta(tildify(env.configPath));
       }
-      return logTasks(tree, opts.depth, function(task) {
+      return logTasks(tree, opts, function(task) {
         return gulpInst.tasks[task].fn;
       });
     }
+    if (opts.tasksJson) {
```

Comment:
Can this addition be moved to a separate commit/PR?  I'm glad we are adding it but I want it to show up in the changelog properly.